vertx-web-api-contract/src/test/resources/swaggers/testOrderSpec.yaml

```diff
@@ -0,0 +1,34 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Vertx test order spec
```

Comment:
Change it to "Test spec for OpenAPI3RouterFactoryTest.loadSpecAndTestPrecedence()"